cve_bin_tool/checkers/expat.py

```diff
@@ -14,7 +14,7 @@
 http://www.cvedetails.com/vulnerability-feed.php?vendor_id=16735&product_id=0&version_id=0&orderby=3&cvssscoremin=0
 
 Easiest way to check CVEs is currently the Changes.txt file.  You can pinpoint the CVEs using grep as follows:
-grep 'Release\|CVE' Changes.txt
+grep 'Release\\|CVE' Changes.txt
```

Comment:
Can you double check that the grep works as expected on the expat changes file after this change, and update it if it's no longer correct?

Reply:
Hello @terriko, Sorry, I probably changed this by a mistake. I'm going to revert this change.

Reply:
I probably changed this because backslash's parsing error  on docstring, if it's parsed with `__doc__` property the grep command would be correct.
Now I understand why docstring signed as regex before. I'm going to revert this too.